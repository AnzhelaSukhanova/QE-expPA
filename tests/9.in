(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and 
 (or (bvult (bvmul x #b0100) (bvmul y #b0101)) (= (bvmul x #b0100) (bvmul y #b0101))) 
 (or (bvult (bvmul y #b1000) (bvmul x #b1010)) (= (bvmul y #b1000) (bvmul x #b1010))))))
