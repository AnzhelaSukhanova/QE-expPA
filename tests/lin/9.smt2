(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(declare-fun z () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and
 (or (bvult x (bvadd (bvmul y #b0011) z)) (= x (bvadd (bvmul y #b0011) z)))
 (or (bvult (bvadd y #b1010) x) (= (bvadd y #b1010) x)))))
