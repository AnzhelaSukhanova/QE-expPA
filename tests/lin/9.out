(not (bvult (bvadd z (bvmul y #b0011)) (bvadd y #b1010)))